templates/web/panels/panel.tt.html

```diff
@@ -54,7 +43,7 @@ <h3 class="panel_title_[% panel.type %]">[% panel.title_element.title %]</h3>
             [% END %]
             <h4 [% IF panel.title_element.type == "grade" %]class="grade_[% panel.title_element.grade %]_title"[% END %]
                 [% IF panel.evaluation.defined %]class="evaluation_[%panel.evaluation %]_title"[% END %]
-                [% IF panel.size == "small" %]style="font-size:1.1rem;"[% END %]
+                <div class="panel[% IF panel.size == 'small' %] small-panel[% END %]">
```

Comment:
There's a syntax issue here:

```suggestion
                [% IF panel.size == 'small' %] small-panel[% END %]"
```